### DISCUSSION TITLE
Update winget package?

### DISCUSSION THREAD

**Discussion by ShayBox** [mitmproxy/mitmproxy#6023] [General]
The winget package is still 2 major versions behind (7.0.4)  
Can this be updated or removed if it's not going to be supported?
https://github.com/microsoft/winget-pkgs/pull/35371

**Comment by mhils** [comment]
Thanks for the heads-up - I didn't know this was a thing. We'll look into getting it removed.

You can get the latest release via winget from the Microsoft Store:
```powershell
winget install --source msstore mitmproxy
```

**Reply by mhils** [reply]
PR here: https://github.com/microsoft/winget-pkgs/pull/100796

**Reply by mhils** [reply]
...and it's gone! Thanks again for reporting, @ShayBox.
